mri/generators/cartesian.py

```diff
@@ -0,0 +1,94 @@
+"""
+Cartesian Kspace-Generators.
+
+At each iteration, a column of kspace is yielded.
+Transpose your data if you want to work row-wise.
+"""
+
+import numpy as np
+
+from .base import KspaceGeneratorBase
+
+
+class Column2DKspaceGenerator(KspaceGeneratorBase):
+    """k-space Generator, at each step a new column fills the existing k-space."""
+
+    def __init__(self, full_kspace, mask_cols, max_iter=0):
+        mask = np.zeros(full_kspace.shape[-2:])
+
+        def flip2center(mask_cols, center_pos):
```

Comment:
Maybe a better name / documentation? Something that reflects what actually is being done in MRI context. If I am right you are basically sampling from center of k-space and then moving outward slowly (while you flip)

Reply:
You are right.